packages/stryker-api/src/report/ScoreResult.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Represents a score result of a file or directory
+ */
+interface ScoreResult {
```

Comment:
As discussed offline, we shouldn't change it. It holds the `mutationScore` among others, so it shouldn't be `MutationScoreResult`. I couldn't find a better name than `ScoreResult`, so leaving it as is.